pycm/curve.py

```diff
@@ -342,6 +342,46 @@ def __repr__(self) -> str:
         """Representation method."""
         return "pycm.RCurve(classes: " + str(self.classes) + ")"
 
+class F1Curve(Curve):
+    """
+    Recall-Curve class.
```

Comment:
F1-Curve class

Reply:
Fixed in [40a0e9a](https://github.com/sepandhaghighi/pycm/pull/621/commits/40a0e9a5a646959ff31ca4582b16cc4c6c23afae)